calc_bc.html

```diff
@@ -576,24 +576,33 @@
                         </div>
                     </div>
                     <div class="btn-group gen-specific g2 g3 g4 g5 g6">
-                        <div class="left" title="Has this Pokemon been revealed with Foresight or Odor Sleuth?">
+                        <div class="left" title="Has this Pok&eacute;mon been revealed with Foresight or Odor Sleuth?">
                             <input class="btn-input" type="checkbox" id="foresightL" />
                             <label class="btn btn-wide" for="foresightL">Foresight</label>
                         </div>
-                        <div class="right" title="Has this Pokemon been revealed with Foresight or Odor Sleuth?">
+                        <div class="right" title="Has this Pok&eacute;mon been revealed with Foresight or Odor Sleuth?">
                             <input class="btn-input" type="checkbox" id="foresightR" />
                             <label class="btn btn-wide" for="foresightR">Foresight</label>
                         </div>
                     </div>
                     <div class="btn-group gen-specific g3 g4 g5 g6">
-                        <div class="left" title="Has this Pokemon's power been boosted by an ally's Helping Hand?">
+                        <div class="left" title="Has this Pok&eacute;mon's power been boosted by an ally's Helping Hand?">
                             <input class="btn-input" type="checkbox" id="helpingHandL" />
                             <label class="btn btn-xxwide" for="helpingHandL">Helping Hand</label>
                         </div>
-                        <div class="right" title="Has this Pokemon's power been boosted by an ally's Helping Hand?">
+                        <div class="right" title="Has this Pok&eacute;mon's power been boosted by an ally's Helping Hand?">
                             <input class="btn-input" type="checkbox" id="helpingHandR" />
                             <label class="btn btn-xxwide" for="helpingHandR">Helping Hand</label>
                         </div>
+                    <div class="btn-group gen-specific g5 g6">
```

Comment:
You didn't close the `<div class="btn-group gen-specific g3 g4 g5 g6">` directly above this block.